src/features/product/apis/ProductsAPI.ts

```diff
@@ -0,0 +1,33 @@
+import { httpClient } from "../../../apis/httpClient";
+import { Product } from "../../../apis/types/response";
+import { CategoryOptionsKey, SortOptionsKey } from "../config/filter";
+
+const ERROR_MESSAGE = "상품 데이터를 가져오는 데 실패했습니다.";
+
+const ENDPOINT = "products";
+
+const sortOptionsMap: Record<SortOptionsKey, string> = {
+  "낮은 가격 순": "price,asc",
+  "높은 가격 순": "price,desc",
+} as const;
+
+export const ProductsAPI = {
+  get: async (
+    category: CategoryOptionsKey,
+    sortOption: SortOptionsKey
+  ): Promise<Product[]> => {
+    const params: Record<string, string> = { page: "0", size: "20" };
+
+    if (category !== "전체") params.category = category;
+    if (sortOption && sortOptionsMap[sortOption]) {
+      params.sort = sortOptionsMap[sortOption];
+    }
+
+    const response = await httpClient.get(
+      `${ENDPOINT}?${new URLSearchParams(params).toString()}`
+    );
```

Comment:
```suggestion
    const response = await httpClient.get(ENDPOINT, { query: params });
```
이렇게 사용할 수 있도록 httpClient 내부를 추상화해주면 어떨까요?